src/amberscript/parser.cc

```diff
@@ -1386,6 +1386,8 @@ Result Parser::ParseExpect() {
     return Result("missing buffer name between EXPECT and EQ_BUFFER");
   if (token->AsString() == "RMSE_BUFFER")
     return Result("missing buffer name between EXPECT and RMSE_BUFFER");
+  if (token->AsString() == "EMD_BUFFER")
```

Comment:
Could we also change "emd" to "hist emd" (or "histogram emd" where appropriate, such as in function names)

Reply:
What does EMD stand for? My preference would be the full histogram over hist.

Reply:
Earth mover's distance. Yeah histogram sounds good.